lib/matplotlib/markers.py

```diff
@@ -359,8 +359,8 @@ def _set_marker(self, marker):
                 Path(marker)
                 self._marker_function = self._set_vertices
             except ValueError as err:
-                raise ValueError('Unrecognized marker style {!r}'
-                                 .format(marker)) from err
+                raise ValueError('Unrecognized marker style '
+                                 f'{marker!r}') from err
```

Comment:
Would prefer a little bit more to join the string:
```suggestion
                raise ValueError(
		       f'Unrecognized marker style {marker!r}') from err
```